mozilla/activity-stream
# Bug 159450 - Update various JavaScript dependencies for Activity Stream

Updates the majority of outdated JavaScript dependencies.  The remainder will be spun out into different bugs.

My current thinking is to rebase this onto master and land all the commits unsquashed, so that we can bisect if there are problems.  I don't love that there's already a merge commit on this branch, but rebasing a series of package-lock.json changes is painful, so I'd prefer to leave it there.  I don't _think_ that this will cause any problems for master, but if it does, I'd love for @k88hudson or @Mardak to tell me I'm wrong about that, rather than finding out the hard way...

The vast majority of changes here are small bug-fixes and enhancements.  The one exception here is that raw-loader has dropped CommonJS support in favor of ES modules, but the messaging tests that use it to load .md files are still passing, so this doesn't appear to be a problem.

To test:

* run automated tests and make sure they pass
* run `./bin/strings-import.js` and inspect the output (the execution appears to be faster!)
